tests/unit/nupic/frameworks/opf/clamodel_test.py

```diff
@@ -82,7 +83,104 @@ def testTemporalAnomalyModelFactory(self):
     Temporal Anomaly configuration will return a model that can return
     inferences
     """
-    modelConfig = (
+
+    inferenceArgs = {u'inputPredictedField': u'auto',
+                     u'predictedField': u'c1',
+                     u'predictionSteps': [1]}
+
+    data = [
+      {'_category': None,
+       '_reset': 0,
+       '_sequenceId': 0,
+       '_timestamp': datetime.datetime(2013, 12, 5, 0, 0),
+       '_timestampRecordIdx': None,
+       u'c0': datetime.datetime(2013, 12, 5, 0, 0),
+       u'c1': 5.0},
+      {'_category': None,
+       '_reset': 0,
+       '_sequenceId': 0,
+       '_timestamp': datetime.datetime(2013, 12, 6, 0, 0),
+       '_timestampRecordIdx': None,
+       u'c0': datetime.datetime(2013, 12, 6, 0, 0),
+       u'c1': 6.0},
+      {'_category': None,
+       '_reset': 0,
+       '_sequenceId': 0,
+       '_timestamp': datetime.datetime(2013, 12, 7, 0, 0),
+       '_timestampRecordIdx': None,
+       u'c0': datetime.datetime(2013, 12, 7, 0, 0),
+       u'c1': 7.0}
+    ]
+
+    model = ModelFactory.create(modelConfig=modelConfig)
+    model.enableLearning()
+    model.enableInference(inferenceArgs)
+
+    for row in data:
```

Comment:
Do the second and third data rows add any test coverage or does all of the testing value come from the first one?